templates/manageCourse.mustache

```diff
@@ -10,7 +10,7 @@
 
     <link rel="stylesheet" type="text/css" href="stylesheets/manageCourse.css" />
     <script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/jquery/1.9.0/jquery.min.js"></script>
-    <script src="/javascripts/controllers/manageCourse.js"></script>
+    <!-- <script src="/javascripts/controllers/manageCourse.js"></script> -->
```

Comment:
Reason for doing this?

Reply:
Testing purpose. We were trying to figure out the dependencies of each js files. This part can be uncommented from my current understanding.

Reply:
Resolved.